packages/components/src/Image/tests/chromatic/SvgImage.stories.tsx

```diff
@@ -0,0 +1,53 @@
+
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { Inline } from "../../../layout/index.ts";
+import { NoResults } from "../../assets/index.ts";
+import { SvgImage } from "../../src/SvgImage.tsx";
+
+const meta = {
+    title: "Components/SvgImage",
+    component: SvgImage,
+    parameters: {
+        chromatic: {
+            delay: 100
+        }
+    }
+} as Meta<typeof SvgImage>;
```

Comment:
`as Meta` should instead be `satisfies Meta`

There is a slack thread that details why this is relevant here: https://workleap.slack.com/archives/C05LE4QPHKL/p1708107373983879